pkg/datapath/linux/datapath.go

```diff
@@ -27,17 +29,22 @@ type linuxDatapath struct {
 	config         DatapathConfiguration
 	loader         *loader.Loader
 	wgAgent        datapath.WireguardAgent
+	lbmap          lbmap.LBMap
 }
 
 // NewDatapath creates a new Linux datapath
 func NewDatapath(cfg DatapathConfiguration, ruleManager datapath.IptablesManager, wgAgent datapath.WireguardAgent) datapath.Datapath {
+	maglev := option.Config.NodePortAlg == option.NodePortAlgMaglev
+	maglevTableSize := option.Config.MaglevTableSize
```

Comment:
We are leaking service internal configs to `pkg/datapath`. This should be in the lbmap constructor.